json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
-# OCP Hardware Validation Spec Draft 0.1
+# OCP Test and Validation Output Specification
 
-## Specs
-`results_spec.json`: this is the main output schema file, with the objects that should implement by any user of the specs. There are restricted to what is strictly need to identify a validation session, trying to keep them as simple as possible and with more optional parameters.
+---
 
-`extensions`: this directory contains a lot of other objects to include in the validation output to make it more detailed and prone to statistics
+# TODO: revision below? (Siamak)
 
-`extensions/artifact_extensions.json`: <Google provide description please>
+---
 
-`extensions/diagnosis.json`: <Google provide description please>
+### Version 2.0
```

Comment:
Maybe move this above the horizontal rule for readability?

Reply:
done. please check/resolve